scripts/zones/Eastern_Adoulin/npcs/Inthius.lua

```diff
@@ -0,0 +1,22 @@
+-----------------------------------
+-- Area: Eastern Adoulin
+-- NPC: Inthius
+-- Type: Weather Reporter
+-- !pos -97.3680 0.0000 -34.3807
```

Comment:
From `npc_list.sql`: use these values (and the zone number):

`!pos -97.145 0.001 -36.710 257`

The intention for _all_ of these comments is that you can copy/paste them into the game and travel to that NPC from any zone.

Reply:
I think best practice is just copy paste form the npc list, because otherwise someone might think it's the npc list that's wrong and try to "fix it" since everywhere else these have always matched.